### ISSUE TITLE
Uncaught Error: connect ETIMEDOUT & Uncaught Error: socket hang up

### ISSUE THREAD

**Issue by yvan-sraka** [atom/atom#5142]
**Atom Version**: 0.170.0
**System**: linux 3.16.0-4-amd64
**Thrown From**: Atom Core**

The bug seems pretty spontaneous, I did not use atom when the error occurred.
The message box `Create issue on atom/atom` appeared a lot of times.

### Stack Trace

Uncaught Error: connect ETIMEDOUT

```
At stream.js:94

Error: connect ETIMEDOUT
  at exports._errnoException (util.js:746:11)
  at Object.afterConnect [as oncomplete] (net.js:990:19)

```

If I scroll a little bit on the list of message boxes, I see a second bug which also occurs many times:

### Stack Trace

Uncaught Error: socket hang up

```
At events.js:85

Error: socket hang up
  at createHangUpError (_http_client.js:214:15)
  at TLSSocket.socketCloseListener (_http_client.js:246:23)
  at TLSSocket.emit (events.js:129:20)
  at TCP.close (net.js:469:12)

```

The commands preceding the two issues are the same:

### Commands

```
  3x -4:03.3 beautify:beautify-editor (atom-text-editor.editor)
  2x -2:24.0 core:save (atom-text-editor.editor)
     -2:07.7 beautify:beautify-editor (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -2:05.4 core:save (atom-text-editor.editor)
     -0:29.4 beautify:beautify-editor (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
  2x -0:25.3 core:save (atom-text-editor.editor)
     -0:18.1 beautify:beautify-editor (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -0:10.1 core:save (atom-text-editor.editor)
  2x -0:00.0 beautify:beautify-editor (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "autoclose-html",
      "language-coffee-script",
      "language-python-django-templates",
      "autocomplete-plus"
    ]
  },
  "editor": {
    "fontSize": 13,
    "showInvisibles": true,
    "tabLength": 4,
    "softWrap": true,
    "invisibles": {}
  }
}
```

### Installed Packages

```coffee
# User
atom-beautify, v0.21.2
color-picker, v1.2.6
file-icons, v1.4.5
html-entities, v0.2.0
linter, v0.10.0
linter-bootlint, v0.0.3
linter-clang, v2.10.0
linter-clojure, v0.0.4
linter-codscriptizer, v0.2.0
linter-coffeelint, v0.1.7
linter-csslint, v0.0.11
linter-dartanalyzer, v0.3.1
linter-elixirc, v0.2.1
linter-erlc, v0.2.0
linter-flexpmd, v0.1.9
linter-harbour, v1.6.0
linter-hlint, v0.3.1
linter-htmlhint, v0.0.8
linter-javac, v0.1.3
linter-js-yaml, v0.1.3
linter-jshint, v0.1.0
linter-jsxhint, v0.1.0
linter-less, v0.3.1
linter-lsc, v1.1.0
linter-lua, v0.1.3
linter-php, v0.0.11
linter-puppet-lint, v0.2.3
linter-pyflakes, v0.0.4
linter-pylint, v0.2.0
linter-rubocop, v0.2.1
linter-rust, v0.0.3
linter-scalac, v0.3.3
linter-scss-lint, v0.0.11
linter-shellcheck, v0.0.6
linter-squirrel, v0.3.0
linter-xmllint, v0.0.5
minimap, v3.5.5
project-manager, v1.14.1
restore-windows, v0.3.2
tabs-to-spaces, v0.8.0
travis-ci-status, v0.11.1

# Dev
No dev packages
```

**Comment by yvan-sraka** [comment]
This issue may be a duplicate of it:

Uncaught Error: socket hang up #5135

**Comment by 50Wliu** [comment]
Please upgrade to the latest Atom version (0.174.0).  This was fixed in 0.173.0.
